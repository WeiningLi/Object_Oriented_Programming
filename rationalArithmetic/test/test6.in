create a 3/2
print a
create b 2/-3
print b
+ a b
- b a
* a b
/ b a
+= a b
-= a b
u- a

