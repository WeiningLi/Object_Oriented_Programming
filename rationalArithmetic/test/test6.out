3/2
-2/3
5/6
-13/6
-1
-4/9
5/6
3/2
-3/2
